From b60caeb936d2368383ecacbaa95bcd6e5a794195 Mon Sep 17 00:00:00 2001
From: He Sheng <hesheng@wxiat.com>
Date: Tue, 7 Jun 2022 17:42:40 +0800
Subject: [PATCH 3037/4585] [Backport] sw64: fix compile error for
 DISCONTIGMEM=y

Sunway inclusion
category: bugfix
bugzilla: https://gitee.com/openeuler/kernel/issues/I56QAM

Reference: https://gitee.com/openeuler/kernel/commit/2cd35d863290741e48b33cfb53f00d233982ffc8

--------------------------------

This option hasn't been tested for a while, and it throws compile
errors because the required include header file is missing.

Signed-off-by: He Sheng <hesheng@wxiat.com>

Signed-off-by: Gu Zitao <guzitao@wxiat.com>
Acked-by: Xie XiuQi <xiexiuqi@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>

Signed-off-by: Wang Hai <wanghai38@huawei.com>
---
 arch/sw_64/include/asm/numa.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/sw_64/include/asm/numa.h b/arch/sw_64/include/asm/numa.h
index 47071007e8ff..4ea8b8de248a 100644
--- a/arch/sw_64/include/asm/numa.h
+++ b/arch/sw_64/include/asm/numa.h
@@ -4,6 +4,7 @@
 #define _ASM_SW64_NUMA_H
 
 #include <linux/nodemask.h>
+#include <asm/memory.h>
 
 #ifdef CONFIG_NUMA
 extern nodemask_t numa_nodes_parsed __initdata;
-- 
2.17.1

